src/routes.py

```diff
@@ -28,11 +37,34 @@ def registration() -> str:
         user.set_password(form.password.data)
         session.add(user)
         session.commit()
+        return redirect(url_for("routes.login"))
     return render_template("registration.html", form=form)
 
 
-@bp.route("/login")
+@bp.route("/login", methods=["POST", "GET"])
 def login() -> str:
```

Comment:
Mypy gives you an error for return type here, because now our may return not only rendered template, but also redirect response. And result of `redirect(...)` is not actually a string - this is a special "response" object.

So to fix these errors in `routes.py` you need to annotate return type like that:
```python
def login() -> str | Response:
```
and this `Response` class can be imported from `werkzeug.wrappers.response` package